### DISCUSSION TITLE
Is this project abandoned?

### DISCUSSION THREAD

**Discussion by lemonlambda** [justinrubek/ayysee#20] [Announcements]
I was wondering if this was abandoned because there hasn't been any updates for a while.

**Comment by justinrubek** [comment]
It isn't entirely abandoned. At least I have plans to continue on it. However my personal life has been really busy the past few months and I have a number of projects that further divide my attention.

Most of the ground work for this is in place, although there are certainly a few bugs, missing language features, and general improvements that could be made. I'd happily review and accept contributions. Otherwise I will eventually get back around to this. I think it could really be a good thing for the Stationeers community if this could get rounded out.

**Comment by justinrubek** [comment]
With the recent major update to the game, I am interested in having this be functional and up to date. I am still quite busy with my life but I am planning to find some time to get this to a decent state.

There have been changes to some of the instructions, new ones, and I can see some slight syntax changes to accommodate this. Luckily the syntax and instructions parts are established enough that it shouldn't be a difficult change. The code generation on the other hand was never fully finished and has some bugs. I'm thinking it may need replaced, but time will tell.
